state/core_access.go

```diff
@@ -156,13 +170,17 @@ func (ca *CoreAccessor) constructSignedTx(
 	return ca.signer.EncodeTx(tx)
 }
 
+// SubmitPayForBlob builds, signs, and synchronously submits a MsgPayForBlob. It blocks until the transaction is committed
+// and returns the TxReponse. If gasLim is set to 0, the method will automatically estimate the gas limit. If the fee is
+// negative, the method will use the nodes min gas price multipled by the gas limit.
 func (ca *CoreAccessor) SubmitPayForBlob(
 	ctx context.Context,
 	fee Int,
 	gasLim uint64,
 	blobs []*blob.Blob,
 ) (*TxResponse, error) {
 	if len(blobs) == 0 {
+		// TODO: publilcly facing errors should be able to be handled
```

Comment:
Do you mean make the error public var?
I agree in general, but in this particular case, it's useless. It's a sanity check.

Reply:
More practically its good for testing

Reply:
Then they should be private.

Reply:
Let's remove the TODO until there is a clear need for such error var